templates/hackmd/0/docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+hackmdPostgres:
+  image: postgres
+  environment:
+    POSTGRES_USER: hackmd
+    POSTGRES_PASSWORD: hackmdpass
+    POSTGRES_DB: hackmd
+
+hackmd:
+  image: hackmdio/hackmd:0.4.5
+  environment:
+    POSTGRES_USER: hackmd
+    POSTGRES_PASSWORD: hackmdpass
+    NODE_ENV: "${NODE_ENV}"
+    DEBUG: "${DEBUG}"
+    HMD_DOMAIN: "${HMD_DOMAIN}"
+    HMD_URL_PATH: "${HMD_URL_PATH}"
+    HMD_PORT: "${HMD_PORT}"
```

Comment:
@elboletaire Can you export this port or create a loadbalancer service to serve the web app for hackmd, thanks.